spec/rails_helper.rb

```diff
@@ -35,7 +35,7 @@
 
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_path = Rails.root.join("spec", "fixtures").to_s
```

Comment:
You might want single quotes here since you removed the interpolation

Reply:
Odd... that was a rubocop auto-fix! I wouldn't have expected it to use double quotes.